metadata/gc.go

```diff
@@ -26,7 +26,7 @@ const (
 
 var (
 	labelGCRoot       = []byte("containerd.io/gc.root")
-	labelGCSnapRef    = []byte("containerd.io/gc.ref.snapshot.")
+	labelGCSnapRef    = []byte("containerd.io/gc.ref.snapshots.")
```

Comment:
This is a breaking change. Let's make sure this gets added to the release notes.

Reply:
Is this a good target for migration? Would be a nice way to test that functionality

Reply:
i can keep it as `snapshot`

Reply:
This should stay as singular, since it is dealing with a singular thing.

Reply:
I agree with Stephen, only 1 snapshot is referenced